server/chart_config/climate.json

```diff
@@ -91,6 +110,53 @@
     },
     "isChoropleth": true
   },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-carbon_dioxide_emissions_nonbiogenic",
+    "title": "Carbon dioxide emissions (non-biogenic)",
+    "description": "Annual amount of carbon dioxide emissions (non-biogenic)",
+    "statsVars": ["Annual_Emissions_CarbonDioxide_NonBiogenic"],
+    "unit": "t",
+    "relatedChart": {
+      "scale": false
+    }
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-methane_emissions_nonbiogenic",
+    "title": "Methane emissions (non-biogenic)",
+    "description": "Annual amount of methane emissions (non-biogenic)",
+    "statsVars": ["Annual_Emissions_Methane_NonBiogenic"],
+    "unit": "MTCO2e",
+    "relatedChart": {
+      "scale": false
+    }
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-nitrous_oxide_emissions_non_biogenic",
+    "title": "Nitrous oxide emissions (non-biogenic)",
+    "description": "Annual amount of nitrous oxide emissions (non-biogenic)",
+    "statsVars": ["Annual_Emissions_NitrousOxide_NonBiogenic"],
+    "unit": "MTCO2e",
+    "relatedChart": {
+      "scale": false
+    }
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-water_withdrawal_rate",
+    "title": "Water withdrawal rate",
+    "description": "Water withdrawal rate",
+    "statsVars": ["WithdrawalRate_Water"],
+    "unit": "mgd",
+    "relatedChart": {
+      "scale": true,
+      "denominator": "Count_Pereson",
```

Comment:
typo: Count_Person

But, I wonder if we should really do per-capita for Water withdrawal from Earth?  Because it is not really a function of people in the state (consider it can flow into other states too)

Reply:
Removed denominator